redbot/cogs/general/general.py

```diff
@@ -347,23 +347,27 @@ def _bitsize(num: int):
 
             features = {
                 "ANIMATED_ICON": _("Animated Icon"),
-                "BANNER": _("Banner Image"),
+                "ANIMATED_BANNER": _("Animated Banner"),
+                "BANNER": _("Banner"),
                 "COMMERCE": _("Commerce"),
                 "COMMUNITY": _("Community"),
-                "DISCOVERABLE": _("Server Discovery"),
+                "DISCOVERABLE": _("Discoverable"),
                 "FEATURABLE": _("Featurable"),
                 "INVITE_SPLASH": _("Splash Invite"),
-                "MEMBER_LIST_DISABLED": _("Member list disabled"),
                 "MEMBER_VERIFICATION_GATE_ENABLED": _("Membership Screening enabled"),
-                "MORE_EMOJI": _("More Emojis"),
-                "NEWS": _("News Channels"),
+                "MONETIZATION_ENABLED": _("Monetization Enabled"),
+                "MORE_STICKIES": _("More Stickies"),
```

Comment:
This one is `MORE_STICKERS`, so "More Stickers". (even tho stickies cute)

Reply:
Lmao thanks. i didnt even notice i typed that.